modules/users/client/views/settings/change-profile-picture.client.view.html

```diff
@@ -3,7 +3,7 @@
     <form class="signin form-horizontal">
       <fieldset>
         <div class="form-group text-center">
-          <img ngf-src="vm.fileSelected ? picFile : '/' + vm.user.profileImageURL" alt="{{vm.user.displayName}}" class="img-thumbnail user-profile-picture" ngf-drop>
+          <img ngf-src="vm.fileSelected ? picFile : vm.user.profileImageURL" alt="{{vm.user.displayName}}" class="img-thumbnail user-profile-picture" ngf-drop>
```

Comment:
You can add a config setting to the [Shared config](https://github.com/meanjs/mean/blob/master/config/env/default.js#L58) that would allow you to check whether the profile should be retrieved using local storage.

Then you wouldn't need remove the "/", thus solving path issues on the backend.

```javascript
shared: {
    profileImageIsExternal: process.env.UPLOADS_STORAGE ? true : false
    ...
}
```